ract/cli.py

```diff
@@ -33,14 +36,16 @@ def ract(cube, art, fps, seconds):
     cube.setup(fps)
 
     start = time.time()
-    # timer = time.time()
+    _timer = time.time()
```

Comment:
Not sure why `_timer` starts with an underscore -- it's a local variable so there's no need to prefix it with an `_` unless is wasn't being used.